CMakeLists.txt

```diff
@@ -79,6 +90,14 @@ set_project_c_warnings(project_c_warnings)
 add_library(project_cpp_warnings INTERFACE)
 set_project_cpp_warnings(project_cpp_warnings)
 
+if(EMSCRIPTEN)
+   #same as project_c_warnings but without -O3. In the future we should
+   #experiment and switch to O3. because it reduces the js output size
+   #significantly.
+   set(CMAKE_C_FLAGS_DEBUG "-g -O0-DNDEBUG -Wall -Wextra -Wpedantic -std=gnu11")
```

Comment:
`-DNDEBUG` is meant to be defined in debug mode, isn't?

Reply:
yep that was going for the release